src/themes/twitter/elements/emoji.variables

```diff
@@ -3259,18 +3259,6 @@
   1f565: clock1030;
   1f566: clock1130;
   1f567: clock1230;
-  30: digit_zero;
-  31: digit_one;
-  32: digit_two;
-  33: digit_three;
-  34: digit_four;
-  35: digit_five;
-  36: digit_six;
-  37: digit_seven;
-  38: digit_eight;
-  39: digit_nine;
-  23: pound_symbol;
-  2a: asterisk_symbol;
```

Comment:
Instead of removing i suggest to reuse the values from the non digit_ entries as it was before 2.9.0

```less
  30-20e3: digit_zero;
  31-20e3: digit_one;
  32-20e3: digit_two;
  33-20e3: digit_three;
  34-20e3: digit_four;
  35-20e3: digit_five;
  36-20e3: digit_six;
  37-20e3: digit_seven;
  38-20e3: digit_eight;
  39-20e3: digit_nine;
  23-20e3: hash_symbol;
  2a-20e3: asterisk_symbol;

```

I also saw, that "pound_symbol" is actually a "hash_symbol"  (23-20e3)

Reply:
it is a map, how can I add 2 names for one key? is the small BC worth it?

Reply:
uhm...you've got a valid point here, my bad.
Ok then lets use the PR as you intentially did it